src/details/ArborX_DetailsDistributedTreeImpl.hpp

```diff
@@ -42,6 +42,34 @@ struct DefaultCallbackWithRank
   }
 };
 
+KOKKOS_INLINE_FUNCTION Sphere buildSphere(Point const &center, float radius)
+{
+  return Sphere{center, radius};
+}
+
+KOKKOS_INLINE_FUNCTION Sphere buildSphere(Box const &box, float distance)
```

Comment:
Why "distance" rather than "radius"?

Reply:
Because the radius of the sphere is this distance + the distance from the center of box to a vertex.